src/controllers/identity/policy/index.ts

```diff
@@ -28,28 +28,46 @@ const getPolicy = async (params) => {
     return response;
 };
 
-const listPolicies = async (params) => {
-    const identityV1 = await grpcClient.get('identity', 'v1');
-    const identityResponse = await identityV1.Policy.list(params);
 
-    /*1. Collect all repository_id
-    2. Collect repository.Policy with the set of repository_id
-    3. Merge repository.Policy into identity.Policy*/
+const listIdentityPolicy = async (params) => {
+    const identityV1 = await grpcClient.get('identity', 'v1');
+    const { results, total_count } = await identityV1.Policy.list(params);
+    const _results = results.map((d) => ({
+        ...d,
+        policy_type: 'CUSTOM'
+    }));
+    return { results: _results, total_count };
+};
+const listRepositoryPolicy = async (params) => {
+    /**
+     * 1. Collect all repository_id
+     * 2. Collect repository.Policy with the set of repository_id
+     * 3. Merge repository.Policy into identity.Policy
+    */
     const repositoryV1 = await grpcClient.get('repository', 'v1');
-    const repositories = await repositoryV1.Repository.list(params);
-    const repositoryPromise = repositories.results.map(d => (repositoryV1.Policy.list({ ...params, repository_id: d.repository_id })));
-    const repositoryResponse = await Promise.allSettled(repositoryPromise);
-
-    const policies = [...identityResponse.results];
-    repositoryResponse.forEach(d => {
-        if (d.status === 'fulfilled') {
-            d.value.results.forEach(policy => policies.push(policy));
+    const { results: repositoryResults } = await repositoryV1.Repository.list(params);
+    const repositoryResponse = await Promise.allSettled(repositoryResults.map(d => (
+        repositoryV1.Policy.list({ ...params, repository_id: d.repository_id })
+    )));
+    let results = [];
+    repositoryResponse.forEach((res) => {
+        if (res.status === 'fulfilled') {
+            const _results = res.value.results.map((r) => ({
+                ...r,
+                policy_type: 'MANAGED'
+            }));
+            results = results.concat(_results);
         }
     });
-
+    return { results, total_count: results.length };
+};
+const listPolicies = async (params) => {
+    const identityResponse = await listIdentityPolicy(params);
+    const repositoryResponse = await listRepositoryPolicy(params);
+    const policyResults = [...identityResponse.results, ...repositoryResponse.results];
```

Comment:
```suggestion
    const policyResults = repositoryResponse.results.concat(identityResponse.results)
```

1. 가능한 concat 써주세요!
2. managed 타입이 먼저 와야합니다!